src/main/java/github/scarsz/discordsrv/listeners/DiscordChatListener.java

```diff
@@ -356,4 +357,22 @@ private boolean processConsoleCommand(GuildMessageReceivedEvent event, String me
         return true;
     }
 
+    private boolean processPlayerCommand(GuildMessageReceivedEvent event, String message) {
+        if (!DiscordSRV.config().getBoolean("DiscordChatChannelPlayerCommandEnabled")) return false;
+
+        String prefix = DiscordSRV.config().getString("DiscordChatChannelPlayerCommandPrefix");
+        if (!StringUtils.startsWithIgnoreCase(message, prefix)) return false;
+        String command = message.substring(prefix.length(), message.length()).trim();
+
+        UUID playerUuid = DiscordSRV.getPlugin().getAccountLinkManager().getUuid(event.getAuthor().getId());
+        if (playerUuid == null) return true;
+
+        Player player = Bukkit.getPlayer(playerUuid);
+        if (player == null) return true;
+
+        Bukkit.getScheduler().runTask(DiscordSRV.getPlugin(), () -> Bukkit.getServer().dispatchCommand(player, command));
+
```

Comment:
Should have a configurable, disalabale (check if string is empty) and expirable message (much like the console commands)